include/openenclave/bits/eeid.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Open Enclave SDK contributors.
```

Comment:
The other files seem to be SGX-only but this file has a name and is in a directory that is not SGX-specific.
I can't tell if this is SGX only (in which case this file is in the wrong directory) or whether changes to other files are also needed to provide non-SGX support.

Reply:
Conceptually this idea does not depend on SGX, but the implementation currently doesn't support anything else, mainly because I assume that the sigstruct is saved in the EEID data.